packages/components/ovh-shell/src/plugin/ux/components/modal.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Vanilla JS Modal compatible with Bootstrap
+ * Inspired by https://github.com/KaneCohen/modal-vanilla
+ */
+const templates = {
+  container: '<div class="modal"></div>',
+  dialog: '<div class="modal-dialog"></div>',
+  content: '<div class="modal-content oui-modal oui-modal_shadow"></div>',
+  header: '<div class="modal-header"></div>',
+  headerClose:
+    '<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>',
```

Comment:
Uhm, i think it should, but i don't think it's very relevant here for now, because this component has been created because we do no yet have a library of components that we can use, so it will be replaced eventually but whatever our new design system offers. 

But yes, ideally, for accessibility issues, it should be translated.